tests/includes/juju.sh

```diff
@@ -218,30 +217,53 @@ juju_bootstrap() {
 		;;
 	esac
 
-	image_metadata=""
+	pre_bootstrap
+
+	# When double quotes are added to ${series}, the juju bootstrap
+	# command looks correct, and works outside of the harness, but
+	# does not run, goes directly to cleanup.
+	command="juju bootstrap ${series} --build-agent=${BUILD_AGENT} ${cloud} ${name} -d ${model} ${BOOTSTRAP_ADD_ON}"
```

Comment:
As this is bash, I think what you would want is:

command="juju bootstrap \"${series}\" ..."
otherwise the " ends up exiting the string, and you can't do:
command='juju bootstrap "${series}"...'
because then none of the variables are expanded.
I'm a little surprised we're naming the controller just based on the series and not something like ${series}-$RANDOM

Reply:
Also, do we run these scripts with bash's 'set -u', which means we need to make sure BOOTSTRAP_ADD_ON has some value (even just the empty string), or it will fail on non-vsphere.

Reply:
Name is the name of the controller, which ends up as ctrl-xxxxxxxx.

Ensured that BOOTSTRAP_ADDITIONAL_ARGS is always set.

Removed the comment @ 222-224 as it no longer applies.